docs/deployment/deploy-without-docker/Load-Sample-Cancer-Study.md

```diff
@@ -26,7 +38,7 @@ export PORTAL_HOME=/Users/johndoe/cbioportal
 The sample gene panel has to be imported before gene panel study data can be added to the database.
 
 ```
-cd <your_cbioportal_dir>/core/src/main/scripts
+cd src/main/resources/scripts 
```

Comment:
below we refer to `cbioportal-core/src/...` but here the `cbioportal-core` root folder is not specified. Is that on purpose?

Reply:
good catch